pkg/pillar/cmd/verifier/verifier.go

```diff
@@ -497,79 +312,6 @@ func handleInitMarkedDeletePendingObjects(ctx *verifierContext) {
 	}
 }
 
-// Create the object download directories we own
-func createDownloadDirs(objTypes []string) {
-
-	workingDirTypes := []string{"verifier", "verified"}
-
-	// now create the download dirs
-	for _, objType := range objTypes {
-		for _, dirType := range workingDirTypes {
-			dirName := types.DownloadDirname + "/" + objType + "/" + dirType
-			if _, err := os.Stat(dirName); err != nil {
-				log.Debugf("Create %s", dirName)
-				if err := os.MkdirAll(dirName, 0700); err != nil {
```

Comment:
Where do the directories get created? 

And you must test by onboarding a fresh device, e.g., using make run to get a qemu device up, then onboard it, to make sure it works correctly when /persist starts empty.

Reply:
You are right again; verifier has to own both `verifier/` and `verified/`, create them, etc. But volumemgr cannot know where these are or rely on them; they are private inside verifier package.

Reply:
Earlier I created all these dir in VolumeMgr. 
Moved it back to Verifier as those directories are owned by Verifier.